tests/_util/test_functions.py

```diff
@@ -31,10 +31,9 @@ def test_get_object_id_from_data_concepts_id_is_none():
         name='test',
         bounds=RealBounds(0.0, 1.0, '')
     )
-    template.uids = {'id': None}
 
     with pytest.raises(ValueError):
-        get_object_id(template)
+        template.uids = {'id': None}
```

Comment:
It makes more sense to me that there would be a ValueError on attempting to set this value to something invalid than waiting till the user attempts to get that value